core/jobs/batch_jobs/exp_title_validation_jobs.py

```diff
@@ -0,0 +1,89 @@
+
+# coding: utf-8
+#
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Validation Jobs for title exploration"""
+
+from __future__ import annotations
+
+from core.domain import exp_fetchers
+from core.jobs import base_jobs
+from core.jobs.io import ndb_io
+from core.jobs.types import job_run_result
+from core.platform import models
+from core.jobs.transforms import job_result_transforms
+
+import apache_beam as beam
+
+(exp_models, ) = models.Registry.import_models([models.NAMES.exploration])
+
+
+class GetNumberOfExpExceedsMaxTitleLengthJob(base_jobs.JobBase):
+    """Job that returns exploration having title length more than 36."""
```

Comment:
Can you also add a TODO for removing this job after it was fixed?

Reply:
TODO always needs to refer to some issue number.  Please create an issue assigned to yourself and refer to it here.

Reply:
Done